### ISSUE TITLE
Events in Calendar Mode not showing correct time

### ISSUE THREAD

**Issue by juleztb** [totaldebug/atomic-calendar-revive#992]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I'm using atomiccalendarrevive in the calendar mode only.
It shows all events, all-day as well as timed events.
But for timed events it doesn't show the time, instead it shows some weird values. Maybe hexadecimals?

### Expected behaviour

Timed events should show the time when they start as well as the time they end in calendar mode.

### Steps To Reproduce

![acr-bug](https://user-images.githubusercontent.com/36635338/231579678-32276854-be12-4b3e-8979-f7f7cb1b97cb.png)

### Card Version

7.3.0

### Home Assistant Version

2023.4.2

### Configuration

```
type: custom:atomic-calendar-revive
name: Kalender
entities:
  - entity: calendar.xxxxx_googlemail_com
    name: Julian
    color: blue
  - entity: calendar.xxxxx_gmail_com
    name: Agnes
    color: yellow
  - entity: calendar.familie
    name: Familie
    color: red
defaultMode: calendar
firstDayOfWeek: 1
maxDaysToShow: 7
refreshInterval: 60
calShowDescription: true
showLastCalendarWeek: false
disableCalEventLink: true
disableCalLink: true
dateFormat: LL
hoursFormat: de
showLocation: true
showLoader: true
showMultiDay: true
disableCalLocationLink: true
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by marksie1988** [comment]
You have the wrong setting for `hoursFormat: de` remove this and it will work.

**Comment by juleztb** [comment]
Thank you for the fast response!
I removed it but it still shows these seemingly hex-values.

Btw: Everytime I open the config it replaces the last (in my case 3rd, but if I remove that entity, it also removes the 2nd) config with "- null".
It still shows it if I don't save, though.
I just recognized this while trying your fix
Screenshot:
![acr-bug2](https://user-images.githubusercontent.com/36635338/231582883-1c4829c9-4d4d-496b-b09f-c78143487387.png)
